feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -444,6 +473,38 @@ fun TopicIcon(
     )
 }
 
+@Composable
+private fun ShowDialog(
```

Comment:
I suggest renaming this function to `ShowNotificationPermissionDialog` for better clarity.